services/m3aggregator/config/aggregator.go

```diff
@@ -46,6 +46,10 @@ import (
 	"github.com/spaolacci/murmur3"
 )
 
+const (
+	initialBufferSizeGrowthFactor = 2
```

Comment:
Sure, I can make it a bit more conservative, e.g., 1.5x. The number of buffers is actually pretty small (8096) so even allocating 2x we are allocating 8MB more (25MB vs. 17MB)